### DISCUSSION TITLE
Show previous car?

### DISCUSSION THREAD

**Discussion by euphguy** [teslamate-org/teslamate#2909] [Q&A] [answered]
I recently returned my leased Tesla and picked up a new one. The new one immediately was picked up by Teslamate, but my old one also disappeared, even from Grafana. I have just started to learn how to look manually at the database and I see data from my old car. It is still listed in the cars table. How would I go about showing its data in Grafana again?

If it matters, my wife's Model Y is also logging data and none of this affected her car. I see my new Model 3 and her Y in Teslamate and Grafana, and I see all three cars in the database.

EDIT: Solved by changing the name of the old car so it wasn't the same as the new one.

**Comment by euphguy** [comment, marked as answer]
Solved by changing the name of the old car so it wasn't the same as the new one. 

`UPDATE cars SET name = 'OldTesla' WHERE id = 1;`

**Comment by siempie76** [comment]
@euphguy Where and how did you change this? I named my new car the same as my old car, but doesn't appear in Grafana. It does in Teslamate though.

**Reply by euphguy** [reply]
> @euphguy Where and how did you change this?

I am going to purposefully be slightly vague as I am not very confident in working in the database. You need to connect to the database container first. Then find the id of the car you want to change. Then run the command:

`UPDATE cars SET name='new_name' WHERE id='database_id_for_old_car';`
